### ISSUE TITLE
Collect FPC metrics

### ISSUE THREAD

**Issue by Sparc0** [tynany/junos_exporter#18]
I would like to be able to collect FPC metrics from chassi based Juniper devices
Command to run: show chassis fpc

Parse out
- Temp
- State
- Slot
- CPU util (now,1min,5min,15min)
- DRAM size
- Heap
- Buffer

**Comment by Sparc0** [comment]
RPC command: get-fpc-information
XML output:
```
<rpc-reply xmlns:junos="http://xml.juniper.net/junos/20.4R0/junos">
    <fpc-information xmlns="http://xml.juniper.net/junos/20.4R0/junos-chassis" junos:style="brief">
        <fpc>
            <slot>0</slot>
            <state>Empty</state>
        </fpc>
        <fpc>
            <slot>1</slot>
            <state>Empty</state>
        </fpc>
        <fpc>
            <slot>2</slot>
            <state>Empty</state>
        </fpc>
        <fpc>
            <slot>3</slot>
            <state>Online</state>
            <temperature junos:celsius="41">41</temperature>
            <cpu-total>15</cpu-total>
            <cpu-interrupt>0</cpu-interrupt>
            <cpu-1min-avg>20</cpu-1min-avg>
            <cpu-5min-avg>18</cpu-5min-avg>
            <cpu-15min-avg>17</cpu-15min-avg>
            <memory-dram-size>3584</memory-dram-size>
            <memory-heap-utilization>8</memory-heap-utilization>
            <memory-buffer-utilization>25</memory-buffer-utilization>
        </fpc>
        <fpc>
            <slot>4</slot>
            <state>Online</state>
            <temperature junos:celsius="44">44</temperature>
            <cpu-total>15</cpu-total>
            <cpu-interrupt>0</cpu-interrupt>
            <cpu-1min-avg>14</cpu-1min-avg>
            <cpu-5min-avg>15</cpu-5min-avg>
            <cpu-15min-avg>15</cpu-15min-avg>
            <memory-dram-size>3584</memory-dram-size>
            <memory-heap-utilization>8</memory-heap-utilization>
            <memory-buffer-utilization>25</memory-buffer-utilization>
        </fpc>
        <fpc>
            <slot>5</slot>
            <state>Online</state>
            <temperature junos:celsius="44">44</temperature>
            <cpu-total>13</cpu-total>
            <cpu-interrupt>0</cpu-interrupt>
            <cpu-1min-avg>12</cpu-1min-avg>
            <cpu-5min-avg>13</cpu-5min-avg>
            <cpu-15min-avg>13</cpu-15min-avg>
            <memory-dram-size>3584</memory-dram-size>
            <memory-heap-utilization>7</memory-heap-utilization>
            <memory-buffer-utilization>25</memory-buffer-utilization>
        </fpc>
        <fpc>
            <slot>6</slot>
            <state>Empty</state>
        </fpc>
        <fpc>
            <slot>7</slot>
            <state>Online</state>
            <temperature junos:celsius="43">43</temperature>
            <cpu-total>15</cpu-total>
            <cpu-interrupt>0</cpu-interrupt>
            <cpu-1min-avg>16</cpu-1min-avg>
            <cpu-5min-avg>16</cpu-5min-avg>
            <cpu-15min-avg>15</cpu-15min-avg>
            <memory-dram-size>3584</memory-dram-size>
            <memory-heap-utilization>7</memory-heap-utilization>
            <memory-buffer-utilization>25</memory-buffer-utilization>
        </fpc>
        <fpc>
            <slot>8</slot>
            <state>Online</state>
            <temperature junos:celsius="46">46</temperature>
            <cpu-total>14</cpu-total>
            <cpu-interrupt>0</cpu-interrupt>
            <cpu-1min-avg>17</cpu-1min-avg>
            <cpu-5min-avg>16</cpu-5min-avg>
            <cpu-15min-avg>15</cpu-15min-avg>
            <memory-dram-size>3584</memory-dram-size>
            <memory-heap-utilization>7</memory-heap-utilization>
            <memory-buffer-utilization>25</memory-buffer-utilization>
        </fpc>
        <fpc>
            <slot>9</slot>
            <state>Online</state>
            <temperature junos:celsius="47">47</temperature>
            <cpu-total>14</cpu-total>
            <cpu-interrupt>0</cpu-interrupt>
            <cpu-1min-avg>13</cpu-1min-avg>
            <cpu-5min-avg>14</cpu-5min-avg>
            <cpu-15min-avg>14</cpu-15min-avg>
            <memory-dram-size>3584</memory-dram-size>
            <memory-heap-utilization>7</memory-heap-utilization>
            <memory-buffer-utilization>25</memory-buffer-utilization>
        </fpc>
        <fpc>
            <slot>10</slot>
            <state>Empty</state>
        </fpc>
        <fpc>
            <slot>11</slot>
            <state>Empty</state>
        </fpc>
    </fpc-information>
    <cli>
        <banner>{master}</banner>
    </cli>
</rpc-reply>

```

**Comment by tynany** [comment]
Thanks for the detailed output! The FPC collector has been implemented in [v1.4.0](https://github.com/tynany/junos_exporter/releases/tag/v1.4.0).
